e2e/helpers.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TranslationFunctions } from "@app/i18n/i18n-types"
+import {
```

Comment:
Is there any reason for this file to end with `.spec.ts`? Since its not a test I dont think it should have the `.spec`.